library/koji_tag.py

```diff
@@ -397,6 +400,17 @@ def ensure_tag(session, name, check_mode, inheritance, external_repos,
                 edits[key] = value
                 edit_log.append('%s: changed %s from "%s" to "%s"'
                                 % (name, key, taginfo[key], value))
+        # Arches must be handled specially (Ansible uses a list, API uses
+        # a flat string).
+        if arches:
+            current_arches = []
+            if taginfo['arches']:
+                current_arches = taginfo['arches'].split()
+            if set(arches) != set(current_arches):
```

Comment:
@mizdebsk pointed out that order *does* matter here (it affects `buildSRPMfromSCM`, and the rpmdiff check), so I'll take out this casting to `set()`